bears/js/JSONFormatBear.py

```diff
@@ -33,14 +39,24 @@ def run(self, filename, file,
         :param escape_unicode: Whether or not to escape unicode values using
                                ASCII.
         """
+        if stat(filename).st_size == 0:
```

Comment:
naaah please use `file`:
`if len(file) == 0:`
\+ I would also add a little comment explaining why you explicitly catch this case
\+ I would move this into a separate commit